elements/pfe-footer/README.md

```diff
@@ -0,0 +1,119 @@
+# PFElements Footer Element
+
+`pfe-footer` is our standard Red Hat branded web applications footer.
+
+## Usage
+
+### Minimal Footer Bar
+
+```
+<pfe-footer role="contentinfo">
+  <section slot="pfe-footer-legal">
+    <small>Copyright &copy;2019 Red Hat, Inc.</small>
```

Comment:
Would it make sense to make the year a "smart" year and let JS update it automagically?

Reply:
100%, yes. At the time, I wasn't quite sure the best approach here. I'm open to ideas and anyone who wants to code this "smart" year bit.

Reply:
Just for thoroughness: we want to make this copyright area customizable yet automatically updatable. Having the `<small>Copyright {date} {company name}</small>` as part of the lightDOM with a static date seems like critical lightDOM type stuff. The upgrade to the current year is something JS can/should handle. How do we do that without adding extraneous HTML, like `<span class="date">` to help the JS find it? We might have to, but I'd rather not have to, ya know?

Reply:
I think the span tag is a great idea, otherwise we could do a grep on the light DOM string for a year format and replace that if dynamic date is desired -- should that functionality be opt-in or opt-out for this component?